.gitignore

```diff
@@ -41,3 +41,10 @@ Thumbs.db
 
 # Next.js
 .next
+
+# VsCode
+.vscode
```

Comment:
If I am correct VS Code is covered above at line 21

Reply:
You are correct! I missed that. The reason I readded it is it's trying to commit .vscode/settings.json, that file get's excluded from the ignore at line 21. What do you think, should we include the file or not? It currently contains 

```{
  "cSpell.words": [
    "Yalc"
  ]
}```

Reply:
The solution for this was to remove all exceptions to the .vscode folder from the .gitignore